modules/deploy/pages/redpanda/manual/resilience/shadowing.adoc

```diff
@@ -0,0 +1,672 @@
+  - resource_filter:
+      resource_type: "TOPIC"
+      pattern_type: "PREFIX"
+      name: "prod-"
+    access_filter:
+      principal: "User:app-user"
+      operation: "READ"
+      permission_type: "ALLOW" 
+      host: "*"
+  # Include consumer group permissions
+  - resource_filter:
+      resource_type: "GROUP"
+      pattern_type: "LITERAL"
+      name: "*"
+    access_filter:
+      principal: "User:app-user" 
+      operation: "READ"
+      permission_type: "ALLOW"
+      host: "*"
+----
+
+==== Generate a sample configuration
+
+Use `rpk` to generate a sample configuration file with common filter patterns:
+
+[,bash]
+----
+rpk shadow config generate --output shadow-link-config.yaml
+----
+
+This creates a template configuration file that you can customize for your environment.
+
+=== Networking
+
+Configure network connectivity between your source and shadow clusters to enable shadow link replication. The shadow cluster initiates connections to the source cluster using a pull-based architecture.
+
+==== Connection requirements
+
+* **Direction**: Shadow cluster connects to source cluster (outbound from shadow, inbound to source)
+* **Protocol**: Kafka protocol over TCP (default port 9092, or your configured listener ports)
+* **Persistence**: Connections remain active for continuous replication
+
+==== Firewall configuration
+
+Configure firewall rules to allow the shadow cluster to reach the source cluster:
+
+**On the source cluster network:**
+
+* Allow inbound TCP connections on Kafka listener ports (typically 9092).
+* Allow connections from the shadow cluster's IP addresses or subnets.
+
+**On the shadow cluster network:**
+
+* Allow outbound TCP connections to the source cluster's Kafka listener ports.
+* Ensure DNS resolution works for source cluster hostnames.
+
+==== Bootstrap servers
+
+Specify multiple bootstrap servers in your shadow link configuration for high availability:
+
+[,yaml]
+----
+client_options:
+  bootstrap_servers:
+    - "source-broker-1.example.com:9092"
+    - "source-broker-2.example.com:9092" 
+    - "source-broker-3.example.com:9092"
+----
+
+The shadow cluster uses these addresses to discover all brokers in the source cluster. If one bootstrap server is unavailable, the shadow cluster tries the next one in the list.
+
+==== Network security
+
+For production deployments, secure the network connection between clusters:
+
+**TLS encryption:**
+[,yaml]
+----
+client_options:
+  tls_settings:
+    enabled: <ENABLE_TLS>                # Enable TLS using system trust store, example: true
+    ca_path: "<CA_CERT_PATH>"            # Path to CA certificate, example: "/etc/ssl/certs/ca.crt"
+    # Optional client certificates for mTLS
+    key_path: "<CLIENT_KEY_PATH>"        # Optional: Path to client private key, example: "/etc/ssl/private/client.key"
+    cert_path: "<CLIENT_CERT_PATH>"      # Optional: Path to client certificate, example: "/etc/ssl/certs/client.crt"
+----
+
+**Authentication:**
```

Comment:
Please add a line that tells users that this example is setting up AuthN using SCRAM, or something that introduces the example.